packages/client/src/providers/AuthProvider/index.js

```diff
@@ -37,26 +25,36 @@ const useAuthProvider = () => {
     const [initialized, setInitialized] = useState(false);
 
     const signup = (email, password) => {
-        return firebase.auth().createUserWithEmailAndPassword(email, password);
+        return Auth.createUserWithEmailAndPassword(email, password);
     }
 
     const signin = (email, password) => {
-        return firebase.auth().signInWithEmailAndPassword(email, password);
+        return Auth.signInWithEmailAndPassword(email, password);
     };
 
+
     const signout = () => {
-        firebase.auth().signOut()
+        Auth.signOut()
         .then(() => {
-            setUser(false)
+            setUser(null)
```

Comment:
Ojasw is it working on a proper sign in page. This page exists for debugging purposes.

Reply:
I don't think javascript has an enum. Do you think creating a string constant would be fine?

Reply:
Probably follow this style https://www.sohamkamani.com/blog/2017/08/21/enums-in-javascript/

Reply:
I am just making a string constant since there is not really a way to give an enum a value. Ideally, the second `LOGGED_IN` state should have the value of the user logged in.